modules/tfm/src/zephyr_tfm_ns_interface.c

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
@ioannisg @tejlmand Please comment on the choice of location for this file. It's used here: https://github.com/oyvindronningstad/trusted-firmware-m-zephyr-module/blob/plat-ext-2/CMakeLists.txt#L245